### DISCUSSION TITLE
Multi-output Inverse Problem with Different Anchors for Each Data Type

### DISCUSSION THREAD

**Discussion by XyM6u9** [lululxvi/deepxde#1803] [Q&A]
I am trying to solve an inverse problem with multiple outputs (state variables). However, the data for each type is not observed at the same locations (anchors). How can I set this up in DEEPXDE?

For example, in the demo [Inverse problem for the Lorenz system with exogenous input
](https://deepxde.readthedocs.io/en/latest/demos/pinn_inverse/lorenz.inverse.forced.html), I need to set the data of y0 and y1 in different time stamps np.linspace(0, 3, 200) and np.linspace(0, 10, 100), respectively:

```
observe_y0 = dde.icbc.PointSetBC(np.linspace(0, 3, 200), ob_y[:, 0:1], component=0)
observe_y1 = dde.icbc.PointSetBC(np.linspace(0, 10, 100), ob_y[:, 1:2], component=1)
```

How can I set different anchors for each type of data in the data object, as shown below?
```
data = dde.data.PDE(
    geom,
    Lorenz_system,
    [ic1, ic2, ic3, observe_y0, observe_y1],
    num_domain=400,
    num_boundary=2,
    anchors=????????,
    auxiliary_var_function=ex_func2,
)

```

**Comment by lululxvi** [comment]
`anchors` is not really needed for `dde.icbc.PointSetBC`.
